tests/e2e/Services/Account/AccountBase.php

```diff
@@ -1389,6 +1390,7 @@ public function testCreateSessionWithMagicUrl($data): array
         $this->assertNotEmpty($response['body']);
         $this->assertNotEmpty($response['body']['$id']);
         $this->assertNotEmpty($response['body']['userId']);
+        $this->assertEmpty($response['body']['secret']);
```

Comment:
Is it right to assert Empty here? 

The AccountBase class is used by AccountsCustomClientTest and AccountCustomServerTest. In the first case, the secret should be empty since the tests are executed in the client scope and should not be empty in the latter since its executed in the server scope